From c2dca1bceca5e768c22710fb7298f2905dbd0ac6 Mon Sep 17 00:00:00 2001
From: shucheng <shucheng@bd-apaas.com>
Date: Mon, 10 Apr 2023 23:07:13 +0800
Subject: [PATCH] labels

---
 command/daemon/daemon.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/command/daemon/daemon.go b/command/daemon/daemon.go
index 63c9850..15a8886 100644
--- a/command/daemon/daemon.go
+++ b/command/daemon/daemon.go
@@ -194,6 +194,7 @@ func (c *daemonCommand) run(*kingpin.ParseContext) error {
 		Client:   cli,
 		Dispatch: runner.Run,
 		Filter: &client.Filter{
+			Machine: config.Runner.Name,
 			Kind:    resource.Kind,
 			Type:    resource.Type,
 			OS:      config.Platform.OS,
-- 
2.39.2

